linter_exclusions.yml

```diff
@@ -2818,12 +2818,24 @@ network watcher connection-monitor create:
     endpoint_dest_resource_id:
       rule_exclusions:
       - option_length_too_long
+    endpoint_dest_address_include:
```

Comment:
Better not to modify "linter_exclusions" and use "options_list" instead, e.g.,
https://github.com/Azure/azure-cli/blob/efdecf4142ce23cccfe2169227b806201b87d0b2/src/azure-cli/azure/cli/command_modules/network/_params.py#L500-L502

Reply:
To make this parameter name meaningful and consistent with other parameters, it has to be added in linter_exclusions.
All source endpoints start with - `endpoint_source*` and destination endpoints starts with `endpoint_dest*`. 

Hence, we'd added it in linter_exclusions, so the argument names would be meaningful and not confusing. Let me know if you see any concerns.